### ISSUE TITLE
Can't find TestFlight.ipa

### ISSUE THREAD

**Issue by Thomas20090425** [Ingan121/FSUntether#7]
```
mayuecheng_account@ThomasMacBookPro 111 % sh build.sh

    ___________ __  __      __       __  __             
   / ____/ ___// / / /___  / /____  / /_/ /_  ___  _____
  / /_   \__ \/ / / / __ \/ __/ _ \/ __/ __ \/ _ \/ ___/
 / __/  ___/ / /_/ / / / / /_/  __/ /_/ / / /  __/ /    
/_/    /____/\____/_/ /_/\__/\___/\__/_/ /_/\___/_/     
                      by Ingan121


Fucking Simple Untethered code execution PoC for iOS 15, 16, and 17
Untether is fully supported on iOS 15, 16, and 17 with BFU code execution.
  (AFU code execution is supported on iOS 14)
Unsandboxing method varies per version; please see the options below.

Preparing...
mkdir: /build: Read-only file system
build.sh: line 31: cd: /build: No such file or directory
Please place a DECRYPTED TestFlight IPA in the root of this project (the same folder as this script.)
The file must be named 'TestFlight.ipa' (without quotes)
build.sh: line 36: cd: OLDPWD not set
mayuecheng_account@ThomasMacBookPro 111 % sudo sh buils.sh
sh: buils.sh: No such file or directory
mayuecheng_account@ThomasMacBookPro 111 % sudo sh build.sh

    ___________ __  __      __       __  __             
   / ____/ ___// / / /___  / /____  / /_/ /_  ___  _____
  / /_   \__ \/ / / / __ \/ __/ _ \/ __/ __ \/ _ \/ ___/
 / __/  ___/ / /_/ / / / / /_/  __/ /_/ / / /  __/ /    
/_/    /____/\____/_/ /_/\__/\___/\__/_/ /_/\___/_/     
                      by Ingan121


Fucking Simple Untethered code execution PoC for iOS 15, 16, and 17
Untether is fully supported on iOS 15, 16, and 17 with BFU code execution.
  (AFU code execution is supported on iOS 14)
Unsandboxing method varies per version; please see the options below.

Preparing...
mkdir: /build: Read-only file system
build.sh: line 31: cd: /build: No such file or directory
Please place a DECRYPTED TestFlight IPA in the root of this project (the same folder as this script.)
The file must be named 'TestFlight.ipa' (without quotes)
build.sh: line 36: cd: OLDPWD not set
```

I've tried twice, and yes I did make sure that TestFlight.ipa is in the same directory as build.sh. I've tried two different versions of the decrypted ipas from two of the biggest decrypted ipa websites, no luck
![IMG_1862](https://github.com/Ingan121/FSUntether/assets/68798885/9976e7dd-5487-43ed-b68c-64cbb18b089f)

**Comment by Thomas20090425** [comment]
Update on this issue, I did manage to build choice 3, by modifying the sh file, here is a few things I modified

1. ever single "../file" will cause the file not found error so instead I modified to just "file"
2. selection seems to be broken so I just kept the script for number 3
3. still has no idea what does the directory /build do but hey some how it doesnt require that at all its just there to give me an error

I will be attaching the file I modified below, feel free to take any of the suggestions or just close this ticket!

[script_mod.txt](https://github.com/Ingan121/FSUntether/files/11716236/script_mod.txt)
